ui/Hotkeys.svelte

```diff
@@ -30,12 +30,7 @@
       event.target &&
       (event.target as HTMLInputElement).type === "text";
 
-    if (
-      !hotkeys.areEnabled() ||
-      screen.isLocked() ||
-      hasInputTarget ||
-      event.repeat
-    ) {
+    if (screen.isLocked() || hasInputTarget || event.repeat) {
```

Comment:
Could you backport this and other unrelated changes to reduce the diff?

Reply:
Done.